### ISSUE TITLE
A11y: footer logo link missing alt text

### ISSUE THREAD

**Issue by paolomoz** [hlxsites/bevespi#260]
https://pagespeed.web.dev/analysis/https-franklin-bevespi-com/2ykrg376io?form_factor=mobile

<img width="1080" alt="Screenshot 2023-11-02 at 15 33 00" src="https://github.com/hlxsites/bevespi/assets/111385/bac2cf7d-8402-44dc-a77a-0ac8a8037d9c">
